build.sbt

```diff
@@ -149,89 +153,138 @@ def ignoreUndocumentedPackages(packages: Seq[Seq[java.io.File]]): Seq[Seq[java.i
+        <connection>scm:git:git@github.com:delta-io/delta.git</connection>
+      </scm>
+      <developers>
+        <developer>
+          <id>marmbrus</id>
+          <name>Michael Armbrust</name>
+          <url>https://github.com/marmbrus</url>
+        </developer>
+        <developer>
+          <id>brkyvz</id>
+          <name>Burak Yavuz</name>
+          <url>https://github.com/brkyvz</url>
+        </developer>
+        <developer>
+          <id>jose-torres</id>
+          <name>Jose Torres</name>
+          <url>https://github.com/jose-torres</url>
+        </developer>
+        <developer>
+          <id>liwensun</id>
+          <name>Liwen Sun</name>
+          <url>https://github.com/liwensun</url>
+        </developer>
+        <developer>
+          <id>mukulmurthy</id>
+          <name>Mukul Murthy</name>
+          <url>https://github.com/mukulmurthy</url>
+        </developer>
+        <developer>
+          <id>tdas</id>
+          <name>Tathagata Das</name>
+          <url>https://github.com/tdas</url>
+        </developer>
+        <developer>
+          <id>zsxwing</id>
+          <name>Shixiong Zhu</name>
+          <url>https://github.com/zsxwing</url>
+        </developer>
+      </developers>,
+
+  bintrayOrganization := Some("delta-io"),
+
+  bintrayRepository := "delta"
+)
 
-releaseCrossBuild := true
+// Don't release the root project
+publishArtifact := false
 
-licenses += ("Apache-2.0", url("http://www.apache.org/licenses/LICENSE-2.0"))
-
-pomExtra :=
-  <url>https://delta.io/</url>
-    <scm>
-      <url>git@github.com:delta-io/delta.git</url>
-      <connection>scm:git:git@github.com:delta-io/delta.git</connection>
-    </scm>
-    <developers>
-      <developer>
-        <id>marmbrus</id>
-        <name>Michael Armbrust</name>
-        <url>https://github.com/marmbrus</url>
-      </developer>
-      <developer>
-        <id>brkyvz</id>
-        <name>Burak Yavuz</name>
-        <url>https://github.com/brkyvz</url>
-      </developer>
-      <developer>
-        <id>jose-torres</id>
-        <name>Jose Torres</name>
-        <url>https://github.com/jose-torres</url>
-      </developer>
-      <developer>
-        <id>liwensun</id>
-        <name>Liwen Sun</name>
-        <url>https://github.com/liwensun</url>
-      </developer>
-      <developer>
-        <id>mukulmurthy</id>
-        <name>Mukul Murthy</name>
-        <url>https://github.com/mukulmurthy</url>
-      </developer>
-      <developer>
-        <id>tdas</id>
-        <name>Tathagata Das</name>
-        <url>https://github.com/tdas</url>
-      </developer>
-      <developer>
-        <id>zsxwing</id>
-        <name>Shixiong Zhu</name>
-        <url>https://github.com/zsxwing</url>
-      </developer>
-    </developers>
-
-bintrayOrganization := Some("delta-io")
-
-bintrayRepository := "delta"
+publish := ()
 
-import ReleaseTransformations._
+// Looks like some of release settings should be set for the root project as well.
```

Comment:
inspired from the connectors repo build.sbt - https://github.com/delta-io/connectors/blob/master/build.sbt#L110